app/controllers/foodcoop/ordergroups_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Foodcoop::OrdergroupsController < ApplicationController
   def index
-    @ordergroups = Ordergroup.undeleted.order('name')
+    @ordergroups = Ordergroup.undeleted.sort_by_param(params["sort"] || "name")
```

Comment:
i know that it was my idea to to write it like you did, but what do you think about a) passing `params` directly and b) move the fallback sorting (`"name"`) into the model too?

Reply:
Sure. Things often get clearer once it's glued into code ;-)
I guess a default value for the method parameter makes most sense?